src/app/components/PortraitVideoCarousel/PortraitVideoPromo/index.tsx

```diff
@@ -39,8 +50,37 @@ export default (item: PortraitVideoPromoProps) => {
 
   const hiddenText = `${headline}, ${mediaType}, ${mediaISO8601Duration ? `${durationTranslation}  ${durationSpokenString}, ` : ''}Play ${mediaType}`;
 
+  const adjustedItemPosition = itemPosition + 1;
+  const eventTrackingData = {
+    componentName: `portrait-video-promo-${adjustedItemPosition}`,
+    groupTracker,
+    itemTracker: {
+      type: 'portrait-video-promo',
+      text: headline,
+      position: adjustedItemPosition,
+      ...(momentDuration && { duration: momentDuration.asSeconds() }),
```

Comment:
Does the duration here mean the duration of the media item, or does it mean the duration of the dwell on that component?